tests/Hedgehog.Tests/PropertyTests.fs

```diff
@@ -0,0 +1,21 @@
+﻿module Hedgehog.Tests.PropertyTests
+
+open Hedgehog
+open Expecto
+open TestDsl
+
+let propertyTests = testList "Property tests" [
+    fableIgnore "generated C# list of five elements is not abbreviated in the failure report" <| fun _ ->
+        Expect.throwsC
+            (fun () ->
+                property {
+                    let! xs = Range.singleton 0 |> Gen.int |> Gen.list (Range.singleton 5) |> Gen.map ResizeArray
+                    return Seq.forall ((>) 0) xs
```

Comment:
I think this test can be simplified by replacing `int` with `unit`.

Reply:
Replaced condition with `return false`